packages/react-components/react-teaching-popover/src/components/TeachingPopoverSurface/useTeachingPopoverSurfaceStyles.styles.ts

```diff
@@ -12,7 +12,6 @@ const useStyles = makeStyles({
   root: {
     ...shorthands.padding(tokens.spacingVerticalL, tokens.spacingVerticalL),
     ...shorthands.borderRadius(tokens.borderRadiusXLarge),
-    ...shorthands.borderWidth('0px'),
```

Comment:
This looks like the root cause of the bug, lets remove the other style changes and ensure that the elements sizing or internal positioning isn't changing from border width - if the border width is modifying sizes then box-sizing prop may be incorrect but we shouldn't need to recalc any style sizes

Reply:
I think this should suffice:
![image](https://github.com/microsoft/fluentui/assets/99835933/35516947-f294-4675-9cd0-0b6070b73513)

Reply:
I think it has to be 322 px to account for the 16px padding and 1 px border (shadow). Otherwise if I change it to 288px it'll look like this: 
<img width="390" alt="image" src="https://github.com/microsoft/fluentui/assets/11350529/7c919501-7e89-4462-905a-2c8616495b88">

Reply:
boxSizing: 'content-box' should fix this as it will ignore the padding and other widths, and only enforce an internal size of 288px: 
![image](https://github.com/microsoft/fluentui/assets/99835933/8f67c4cd-21bb-4e97-934c-bfdb4ded99ba)

This will enable users to modify the tokens/padding and not affect the internal width structure.

![image](https://github.com/microsoft/fluentui/assets/99835933/be7542b3-88b0-4297-8277-9a64477f9281)

Reply:
True my mistake